slides-trusted-networks.md

```diff
@@ -0,0 +1,20 @@
+#DNS configuration on explicitly trusted networks
+
+* Client can authenticate the identity of the network (or pre-existing relationship with the network)
```

Comment:
```suggestion
* Client can authenticate the identity of the network (or pre-existing relationship with the network) and has authorized it to override local DNS settings
```

Reply:
OK, "user" might be better than "client".  I think the user is really authorizing the "network", though.  This authorization is specific to one network, not a general authorization for the "client" when it connects to any future network.

Reply:
Agreed. I will update the above line as follows:
Client can authenticate the identity of the network (or pre-existing relationship with the network) and the user has authorized the client to override local DNS settings for a specific network.